Followup PR for host without a owning cluster (6894)

billfitzgerald0120 opened the pull request:
@gmcculloug @bdunne 
Please Review

Fryguy commented:
@billfitzgerald0120 Can you change the commit message and PR title to reflect what this change actually fixes or tests.  In the commity message body, perhaps reference the corresponding upstream PR that this add tests for?

We really shouldn't reference downstream specifics like "5.5.z" unless it's necessary to explain something.

billfitzgerald0120 commented:
Updated title for better explanation of fix. 

Needed to fix the spec test to be consistent with the merged one. 

Removed MiqServer.seed(s) and add stub(s) for :allowed_storages.

Fryguy commented:
@billfitzgerald0120 Please also update the commit message...if you need help someone can probably help you

billfitzgerald0120 commented:
@gmcculloug 
Please Review
